.github/workflows/main.yml

```
@@ -0,0 +1,62 @@
+# This workflow will install Python dependencies, run tests and lint with a
+# variety of Python versions. For more information see:
+# https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: Pytest with Flake8
+
+on: [push, pull_request]
+
+jobs:
+  build:
+
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-latest, macos-latest]  # TODO(76): add windows-latest
+        python-version: ["3.9", "3.10"]
+        numpy_ver: ["latest"]
+        include:
+          - python-version: "3.8"
+            numpy_ver: "1.20.3"
```

```suggestion
            numpy_ver: "1.21"
```
We can update NEP29 minima as of last month.  Or leave this version for now.